ros/humble.v3.17/ros-humble-nav2-mppi-controller/apkbuild_hook.sh

```diff
@@ -0,0 +1,3 @@
+apkbuild_hook() {
+  makedepends="${makedepends} xsimd-dev"
```

Comment:
https://github.com/alpine-ros/rosdistro/blob/979fec97363f104f737f101ab041bed960d905da/rosdep/base.yaml#L8728-L8729
`xsimd` rosdep key should have `xsimd-dev`

Reply:
https://github.com/alpine-ros/rosdistro/pull/194